modules/oe_content_sub_entity/modules/oe_content_sub_entity_document_reference/modules/oe_content_sub_entity_document_reference_group/config/install/field.storage.oe_document_reference.oe_documents.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - media
+    - oe_content_sub_entity_document_reference
```

Comment:
I re-exported after the switch to the submodule, but still, feel this should be oe_content_sub_entity_document_reference_group
opinion ?